Assignment 3.Rmd

```diff
@@ -127,7 +130,7 @@ Now we can plot our line plot using the ggplot command, "ggplot()".
 - Finally we are going to clean up our axes labels: xlab("Week") & ylab("Average Motivation")
 
 ```{r}
-
+ggplot(K6,mapping = aes(x=week,y=avg,color=cluster))+geom_line()+xlab("Week")+ylab("Average Motivation")
 
```

Comment:
You don't need to state the `mapping` , `x` or `y` statements explicitly, you can just write:
`ggplot(K6, aes(week, avg, color=cluster)) + geom_line() + xlab("Week") + ylab("Average Motivation")`